README.md

```diff
@@ -631,14 +646,11 @@ Provide your application useful name to be shown on your phone and select Add
 
 ![PowerApps Home Screen Details](docs-images/powerapps-homescreen-detail.png)
 
-## Congratulations!! 
+## Congratulations
```

Comment:
Don't like exclamation marks? 😆😊

Reply:
Not me but Markdown lint Gods frown upon punctuation marks in heading.

Reply:
Ah ok thats fair - we will get rid of the exclamation marks 😆 I'll keep my excitement to myself 😂